src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center left 25%;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  padding-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(2),
```

Comment:
Use `-n+a` expression instead number.

`a` - number of active stars

This will help make the code more concise

![image](https://user-images.githubusercontent.com/56752562/218985404-ff435a16-0918-40ea-bb2d-20727e1c23c4.png)